runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiSMREntry.java

```diff
@@ -81,6 +89,10 @@ public void setEntry(ILogData entry) {
 
     @Override
     public List<SMREntry> getSMRUpdates(UUID id) {
+        if (trimmed) {
+            return Collections.singletonList(SMREntry.TRIMMED_ENTRY);
```

Comment:
I think we should return empty list here.

Reply:
done.